project/map_set_store/.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
```

Comment:
In general I'd like to see modules such as this named for what they do and not for how they are implemented.  Think about explaining to future readers of the code _why_ this module exists.